diff --git a/esm/manager/locallySpawnedInstance.js b/esm/manager/locallySpawnedInstance.js
index f085dd5ceb9dc8ddcf062d7524d02fc7a408850c..1055532db8b1c077fcbd20319732078d3fb349c7 100644
--- a/esm/manager/locallySpawnedInstance.js
+++ b/esm/manager/locallySpawnedInstance.js
@@ -1,7 +1,8 @@
 import os from 'os';
 import path from 'path';
 import { v4 as uuidv4 } from 'uuid';
-import * as fs from 'fs-extra';
+import fsExtra from 'fs-extra';
+const fs = fsExtra;
 import * as tar from 'tar';
 import { download, sendRequest, sendMultipartRequest } from '../fetch';
 import { retry, sleep } from '../utils';
